clients/wallet/src/stellar_wallet.rs

```diff
@@ -287,15 +287,28 @@ impl StellarWallet {
 		Ok(envelope)
 	}
 
+	/// Sends a 'Payment' Operation transaction specifically for redeem requests.
```

Comment:
I'd rephrase to the following because otherwise it's technically wrong
```suggestion
	/// Sends a 'Payment' transaction specifically for redeem requests.
```